tests/jerry/generators/generate-arguments-2.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+
+file = open("tests/jerry/arguments-2.js", "w")
```

Comment:
Please ensure that this relative path will be OK even is the script is called from a different working directory.
And use path.join, instead of linux path schema.